example/form/CAP_Adrenal.xml

```diff
@@ -19,9 +18,9 @@
     <OtherText name="CS_Version" type="meta" styleClass="right" val="Version 2" />
     <OtherText name="FIGO_Version" type="meta" styleClass="right" />
   </Header>
-  <Body ID="129.1000043_DefaultBodyGUID:_18b9cd14-2d84-4c8f-8bb6-7da6db5f05c6">
+  <Body ID="Body_129.1000043_18b9cd14-2d84-4c8f-8bb6-7da6db5f05c6">

```

Comment:
Body element ID attribute value was invalid (violated the facets of the underlying XSD type, anyURI).